source/transactions/tests/create-collection.yml

```diff
@@ -0,0 +1,131 @@
+runOn:
+  -
+    minServerVersion: "4.3.4"
+    topology: ["replicaset", "sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: explicitly create collection using create command
+
+    operations:
+      - name: dropCollection
+        object: database
+        arguments:
+          collection: *collection_name
+      - name: startTransaction
+        object: session0
+      - name: createCollection
+        object: database
+        arguments:
+          session: session0
+          collection: *collection_name
+      - name: assertCollectionNotExists
+        object: testRunner
+        arguments:
+          database: *database_name
+          collection: *collection_name
+      - name: commitTransaction
+        object: session0
+      - name: assertCollectionExists
+        object: testRunner
+        arguments:
+          database: *database_name
+          collection: *collection_name
```

Comment:
> Note that these currently can't be used along with a session argument as the server does not allow running listCollections and listIndexes commands in a transaction.

I'm not suggesting you change this, but couldn't we technically use `object: session0` _here_ since the session would no longer be in an active transaction?

Reply:
Yes, once the collection is done we could use a `session` argument for the check. I'm not sure if there's an advantage to checking within the session, but I can add it if it makes sense to do so.